web/public/elm-setup.js

```diff
@@ -341,13 +341,10 @@ app.ports.rawHttpRequest.subscribe(function(url) {
     app.ports.rawHttpResponse.send('success');
   });
 
-  xhr.open('GET', url, false);
+  xhr.open('GET', url, true);
```

Comment:
this defaults to true, so we could just remove the third param?

Reply:
hmm i think you are right. although it is now pretty explicit.
its up to you. i don't really have a opinion on it now